packages/angular_devkit/build_facade/src/target.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { BaseException, JsonObject } from '@angular-devkit/core';
+
+
+export class MissingTargetBuildSystemException extends BaseException {
+  constructor(props: JsonObject) {
+    super(`Configuration does not contain or inherit build system: ${props}.`);
+  }
+}
+
+export class Target {
```

Comment:
This should be an interface.

Reply:
It should. I think you mean the `Target` class in `build_facade/src/build-system.ts` though.

Having these two is confusing. These simpler project/target/configuration inside the workspace should have different names or be removed altogether.